CHANGES_tcell.md

```diff
@@ -0,0 +1,56 @@
+# Change from termbox to tcell
+
+Original GOCUI was written on top of [termbox](https://github.com/nsf/termbox-go) package. This document describes changes which were done to be able to use to [tcell/v2](https://github.com/gdamore/tcell) package.
+
+## Attribute color
+
+Attribute type represents a terminal attribute like color and font effects. Color and font effects can be combined using bitwise OR (`|`).
+
+In `termbox` colors were represented by range 1 to 256. `0` was default color which uses the terminal default setting.
+
+In `tcell` colors can be represented in 24bit, and all of them starts from 0. Valid colors have special flag which gives them real value starting from 4294967296. `0` is a default similart to `termbox`.    
+The change to support all these colors was made in a way, that original colors from 1 to 256 are backward compatible and if user has color specified as
+`Attribute(ansicolor+1)` without the valid color flag, it will be translated to `tcell` color by subtracting 1 and making the color valid by adding the flag. This should ensure backward compatibility.
```

Comment:
```suggestion
`gocui.Attribute(ansicolor+1)` without the valid color flag, it will be translated to `tcell` color by subtracting 1 and making the color valid by adding the flag. This should ensure backward compatibility.
```